src/mudlet.cpp

```diff
@@ -3862,15 +3888,18 @@ void mudlet::setShowIconsOnMenu(const Qt::CheckState state)
     }
 }
 
-// during Mudlet load, darktheme will already be set to 'true' -
-// so pass a separate flag to override the state check
-void mudlet::setDarkTheme(const bool& state, const bool& loading)
+void mudlet::setAppearance(const Appearance state, const bool& loading)
 {
-    if (mDarkTheme == state && !loading) {
+    if (state == mAppearance && !loading) {
         return;
     }
 
-    if (state) {
+    mDarkMode = false;
```

Comment:
How about this for something a bit shorter?
```diff
-    mDarkMode = false;
-    if (state == Appearance::dark || (state == Appearance::system && desktopInDarkMode())) {
-        mDarkMode = true;
-    }

-    if (mDarkMode) {
+    if (mDarkMode = ((state == Appearance::dark) || (state == Appearance::system && desktopInDarkMode())), mDarkMode) {
```

Reply:
I think that is more difficult to follow.